lib/inferno/apps/cli/templates/README.md.tt

```diff
@@ -0,0 +1,199 @@
+# <%= title_name %> Test Kit
+
+<%= human_name %> [Inferno](https://github.com/inferno-community/inferno-core) test kit
+for FHIR testing.
+
+## Documentation
+- [Inferno documentation](https://inferno-framework.github.io/inferno-core/)
+- [Ruby API documentation](https://inferno-framework.github.io/inferno-core/docs)
+- [JSON API documentation](https://inferno-framework.github.io/inferno-core/api-docs)
+
+## Instructions for Developing tests
```

Comment:
These details have been moved out of the template readme, so this should be updated to reflect the current state of the readme.

Reply:
fixed, but I made a slight edit to the README mentioning `inferno new`